eclipse-theia/theia-cpp-extensions
# Repo: Uplift to Theia 1.27.0

#### What it does


Closes #149.
This PR updates the repo to use Theia@1.27. 

#### How to test


0. The example application should build and all features should work as expected.
1. Pay special attention to context menus in the Variables view. They've been a bit flaky.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
